generator/js2yaml.html

```diff
@@ -87,6 +89,10 @@ <h1>Convert JS examples into Yaml</h1>
         }
 
         if (Array.isArray(object)) {
+            if (object.length === 0) {
+                return ' []';
+			}
```

Comment:
Heads up: you're using tabs for indentation here and in the diff below. Looks like it should have been spaces.

Reply:
Thanks, fixed.
We may need a `.editorconfig` to have a cross-editor configuration if indentation style.